FormValidator/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Form Validator</title>
+    <style>
+        .main {
+            margin: 0 auto;
+            max-width: 50rem;
+            text-align: center;
+            font-size: large;
+        }
+        h1 {
+            text-align: center;
+        }
+        #main-form input {
+            margin: auto .2rem;
+        }
+
+        #main-form > div {
+            margin-bottom: 1rem;
+        }
+        .result {
+            font-size: xx-large;
+        }
+
+        .text-red {
+            color: red !important;
+        }
+    </style>
+</head>
+<body>
+    <h1> Form Validator</h1>
+    <div class="main">
+        <form id="main-form">
+
+            <div>
+                <label for="email">Email</label>
+                <input type="email" name="email" placeholder="Please input your email" id="email">
+            </div>
+
+            <div>
+                <label for="name">Name</label>
+                <input type="text" name="name" placeholder="Please input your name" id="name">
+            </div>
+
+            <div class="gender-select">
+                <span>Select your gender</span>
+                <div>
+
+                    <label for="gender-male">Male</label>
+                    <input type="radio" name="gender" value="male" id="gender-male">
+                </div>
+                <div>
+                    <label for="gender-female">Female</label>
+                    <input type="radio" name="gender" value="female" id="gender-female">
+                </div>
+            </div>
+
+            <div>
+                <label for="phone">Number</label>
+                <input type="text" name="phone" placeholder="Please input phone number" id="phone">
+            </div>
+            <div>
+                <input type="submit" value="Submit">
```

Comment:
Please replace Input with the Button tag. Button tag is accessible and supported by the assistive technologies